packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+NAW-gegevens zijn bijvoorbeeld een logisch onderwerp die te groeperen is. Ook ondertekening bestaande uit bijvoorbeeld datum, naam en plaats zijn zo’n logische groep.
 
-### Information density
+De daadwerkelijke vragen kunnen tevens één groep zijn, mits deze niet té groot wordt; in dat geval dienen de vragen tevens logisch in onderwerpen opgedeelt te worden.
 
-When structuring a form, there should be enough space between different visual elements. These elements must also be well grouped so that it is clear that, for example, a label belongs to a text input field. Takes different states of elements into account; for example, a possible error message should be in the proximity of the erroneous element and show a direct context.
+### Meerstappen formulieren
 
-If the form is long or contains a lot of information and becomes difficult for users to process. Consider breaking it down into multiple steps or pages: ‘[chunking](https://www.nngroup.com/articles/chunking/)’.
+Als het om een beperkt aantal onderwerpen gaat die onderverdeeld kunnen worden in een klein aantal secties dan kunnen deze op een enkele pagina getoond worden.
 
-## Text labels
+Als het formulier lang is of veel informatie bevat en hierdoor lastig te interpreteren is voor bezoekers, overweeg dan deze op te splitsen in meerdere stappen en/of pagina's. Zorg er echter voor dat deze stappen in betekenisvolle groepen zijn verdeeld.
 
-A text label allows the user to scan a form clearly and comfortably. All form elements must be accompanied by a textual label; a checkbox, select menu, radio button, textarea, etc.
+Als het lastig blijkt het formulier te interpreteren als niet alle vragen op een enkel scherm voorhanden zijn, splits deze dan niet op in meerdere stappen.
 
-A proper label is placed outside of the input field, not within it. This is naturally the case with most form elements; a checkbox, for example, has no room to internally accommodate a text label. For buttons, this label is the button text itself; a button performs an action and is not an element where a user can enter data.
+#### Navigatie en oriëntatie
 
-Several studies have shown that the best position for a text input label is directly top left of an input field (for LTR languages). In this way, the label always has a fixed position and can be displayed in a readable text size. Because of this fixed position, a form can be easily scanned and a user can immediately distinguish between a field that has been filled in and a field that still needs to be filled in.
+Wanneer een formulier uit meerdere stappen en/of pagina’s bestaat dan dient deze voorzien te zijn van navigatie. Deze informeert over het totale aantal stappen, de huidige positie binnen deze stappen, en biedt een manier om hiertussen te navigeren. Bij het gebruik van meerdere pagina’s dient de ingevoerde informatie tussen pagina’s opgeslagen te worden en is deze mogelijk te wijzigen door de bezoeker.
```

Comment:
Moeten we hier specifiek maken dat "hiertussen navigeren" zowel voorwaarts als terug moet ondersteunen voor een volwaardig bruikbaar patroon?

Reply:
Dit gaat op de schop n.a.v. de laatste formulierensessie…